test/benchmark/fuse_db/bin/mysql_bench.py

```
@@ -0,0 +1,45 @@
+#   SCAN : SUM(CRC32(payload)) over the table (reads every payload) -> MB/s
+import os, sys, subprocess, time
+
+sock  = sys.argv[1]
+db    = sys.argv[2]
+rows  = int(sys.argv[3])
+batch = int(sys.argv[4])
+oltp  = int(sys.argv[5])
+MYSQL = os.environ.get("MYSQL_BIN", "/opt/homebrew/opt/mysql/bin/mysql")
+
+def run(sql):
+    r = subprocess.run([MYSQL, "--socket", sock, "-uroot"], input=sql.encode(),
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)